launch-week-august-2023.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Launch Week'
+description: 'Join us for an exciting week filled with announcements, fun activities, and a chance to win prizes!'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/launch-week.jpg
+  alt: 'Launch Week'
+author: Alex Francoeur
+date: 08-28-2023
+published: true
+slug: launch-week-august-2023
+---
+
+The Xata team has been hard at work all summer building the features you’ve been asking for. To commemorate the end of the season, we're launching our latest features in a week-long event. Join us in celebrating our launch week, complete with a content hackathon. Summer is not over yet, let’s enjoy one last party! 🎉 🎊 🥳 🍾
+
+To stay up to date with the latest info, keep an eye on this page and watch for updates on \[[X | Twitter](https://twitter.com/xata)] or [subscribe to our blog](https://xata.io/blog#subscribe-blog). We’ll update our blog every day with the latest announcements.
+
+## Releases
+
+To make things more fun, we’re giving a riddle-like hint for each day. The first day is already revealed, but can you guess what we’re announcing in the other days? If yes, tell us on the social network formally known as Twitter and you’ll get points in our [content hackathon](#xata-content-hackathon-!) (see below).
+
+|           | Hint                                                                                        | Release blog post(s)            |
+| --------- | ------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------| |
+| Monday    | The most popular data format is not necessarily the best. Or is it?                         | Effortlessly [import and export your data](/blog/csv-import-export) using the new CSV implementation in the Xata UI. Sounds simple, but the devil is in the details. Check out the blog post! |
```

Comment:
I dropped a similar nit here as the CSV blog, but I think we could say "in Xata" or "in the Xata app" instead of UI. https://github.com/xataio/mdx-blog/pull/79#discussion_r1307252987

Reply:
Removed the Xata UI mention.